src/components/Form/FileUploadInput.tsx

```diff
@@ -0,0 +1,56 @@
+import { Flex, TextField, Button } from '@radix-ui/themes'
+import { FieldGroup } from './FieldGroup'
+import { Controller, FieldErrors } from 'react-hook-form'
+
+type FileUploadInputProps = {
+  name: string
+  label: string
+  buttonText: string
+  hint?: string
+  errors?: FieldErrors
+  onSelectFile: () => void
+}
+
+export const FileUploadInput = ({
```

Comment:
I extracted this file into its own component so it can be reused for the recorder's Browser Path as well